docs/website/src/main/resources/certifications/jakarta-restful-ws/2.1/TCK-Results.md

```diff
@@ -0,0 +1,35 @@
+TCK Results
+===========
+
+As required by the
+[Eclipse Foundation Technology Compatibility Kit License](https://www.eclipse.org/legal/tck.php),
+following is a summary of the TCK results for releases of Jakarta RESTful Web Services.
+
+# Jersey 2.28, as delivered in Eclipse GlassFish 5.1, Full Profile, Certification Request
+
+- Product Name, Version and download URL (if applicable) \
+  Eclipse GlassFish 5.1, includes: Jakarta RESTful Web Services API 2.1.3 with Jersey 2.28
+  [Eclipse GlassFish 5.1](https://www.eclipse.org/downloads/download.php?file=/glassfish/glassfish-5.1.0.zip)
```

Comment:
These results should be hosted by the Jersey project.

Reply:
done